test-data/earth-mars/networkgraph-5nodes.json

```diff
@@ -29,7 +29,7 @@
         "maximize_send_and_receive_windows": true,
         "max_ack_delay_ms": 18446744073709551615,
         "ack_eliciting_threshold": 10,
-        "use_ecn_based_reno": false
+        "congestion_controller": "ecn_reno"
```

Comment:
_note: this comment applies to other similar files as well_

Interesting... I think you found a bug in our input files.

Setting `use_ecn_based_reno` to `false` gives the false impression that congestion control is disabled, but according to the original Rust code we were actually using `ecn_reno` because we forgot to provide a value for `fixed_congestion_window`!

Is that why you updated the file to use `ecn_reno`? If so, what we actually want is to use `no_cc`, like in `test-data/earth-moon/networkgraph-1orbiter-1moonasset.json`.